app/views/suggestions/_preview.html.erb

```diff
@@ -7,9 +7,9 @@
       </div>
       <div class='media-body'>
         <h4 class='media-heading text-default'><%= suggestion.title %></h4>
-        <p class='media-description'><%= truncate(suggestion.description, length: 250) %></p>
+        <p class='media-description'><%= truncate(suggestion.description.html_safe, length: 250) %></p>
```

Comment:
Probably want to strip tags on this as well: `truncate(strip_tags(suggestion.description).html_safe, length: 250)`